common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -338,6 +359,19 @@ div.problem {
       }
     }
 
+    &.partially-correct, &.ui-icon-check {
+      p.status {
+        display: inline-block;
+        width: 25px;
+        height: 20px;
+        background: url('../images/partially-correct-icon.png') center center no-repeat;
```

Comment:
OK, let me know when you do that so I can update the sandbox (sooner is better, because I'll want to show the sandbox to docs and ux before merging, to verify things are working properly).